packages/actionmenu/src/react/__stories__/index.story.js

```diff
@@ -6,54 +6,7 @@ import React from 'react'
 import Icon from '@pluralsight/ps-design-system-icon/react'
 import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
 
-import ActionMenu from '..'
-
-// TODO: snapshot test generator can't handle.
```

Comment:
There are a couple TODO removals here.  Just removing the backlog that I'm doing nothing about.